src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useRef, useState } from 'react'
+import {
+  Image,
+  StyleSheet,
+  Dimensions,
+  ScrollView,
+  TextInput,
+} from 'react-native'
+
+import Spinner from 'react-native-spinkit'
+
+import { Box, Text, Button } from '@components'
+import { palette } from '@utils/theme'
+
+enum Stages {
+  Welcome = 0,
+  Login = 1,
+  Loading = 2,
+}
+const dimensions = Dimensions.get('screen')
+
+export const LoginScreen = () => {
+  const loginScroll = useRef(null)
+  const [stage, setStage] = useState(0)
+  const [value, onChangeText] = React.useState('')
+
+  // Don't let user continue until they enter correct value
+  const buttonDisabled = stage == Stages.Login && value == ''
+
+  // Increment through each stage
+  const nextStage = () => {
+    let nextStage = stage + 1
+    setStage(nextStage)
+    // Update scorller
+    loginScroll.current?.scrollTo({
+      x: nextStage * dimensions.width,
+      y: 0,
+      animated: true,
+    })
+  }
+
+  return (
+    <Box
+      flex={1}
+      flexGrow={1}
+      height={dimensions.height}
+      flexDirection={{
+        phone: 'column',
+      }}
+      alignItems="center"
+      justifyContent="flex-start"
+      backgroundColor="greenPrimary"
+      paddingVertical={10}
+    >
+      <Text variant="heading" paddingTop={8} paddingBottom={2}>
+        FLARe
+      </Text>
+
+      <Text
+        variant="caption"
+        paddingBottom={4}
+        textAlign="center"
+        paddingHorizontal={6}
+      >
+        Fear Learning and Anxiety Response
+      </Text>
+
+      {/* Animate transition from image to login form to loading screen */}
+      <ScrollView
+        ref={loginScroll}
+        pagingEnabled
+        horizontal
+        scrollEnabled={false}
+        decelerationRate="fast"
+        contentContainerStyle={{ width: `300%` }}
+        showsHorizontalScrollIndicator={false}
+      >
+        {/* Placeholder Image Block */}
+        <Box
+          width={dimensions.width}
+          alignItems="center"
+          justifyContent="flex-start"
+        >
+          <Image
+            source={require('../assets/images/fireworks.png')}
+            style={styles.image}
+          />
+        </Box>
+
+        {/* Login Form Block */}
+        <Box
+          width={dimensions.width}
+          alignItems="center"
+          justifyContent="flex-start"
+          paddingTop={10}
+        >
+          <Box>
+            <Text fontWeight="bold" color='darkGrey' paddingBottom={2}>
+              Participant ID
+            </Text>
+            <TextInput
+              placeholder="Example: ANIXETY-jBSkjbckjb"
+              style={styles.loginField}
+              onChangeText={(text) => onChangeText(text)}
+              value={value}
+            />
```

Comment:
This probably also needs its own component so we can re-use the input styles and build it with restyle instead of stylesheet. Happy to go with this for now until we need it again though.